db/Pos/DB/Class.hs

```diff
@@ -183,13 +183,16 @@ instance {-# OVERLAPPABLE #-}
 -- alternative DBs my be used to provide this data.
 class Monad m => MonadGState m where
     gsAdoptedBVData :: m BlockVersionData
+    gsIsBootstrapEra :: m Bool
```

Comment:
Please apply the suggestions from Slack — this function needs an `EpochIndex` as input.